packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -321,19 +323,14 @@ - (BOOL)setupWriterForPath:(NSString *)path {
   if (path != nil) {
     outputURL = [NSURL fileURLWithPath:path];
   } else {
-    NSArray *paths =
-        NSSearchPathForDirectoriesInDomains(NSDocumentDirectory, NSUserDomainMask, YES);
-    NSString *documentsDirectoryPath = [paths objectAtIndex:0];
-    time_t unixTime = (time_t)[[NSDate date] timeIntervalSince1970];
-    NSString *timestamp = [NSString stringWithFormat:@"%ld", unixTime];
-    NSString *filename = [NSString stringWithFormat:@"iPhoneVideo_%@.mp4", timestamp];
-    outputURL =
-        [NSURL fileURLWithPath:[documentsDirectoryPath stringByAppendingPathComponent:filename]];
+    return NO;
   }
   _videoWriter =
       [[AVAssetWriter alloc] initWithURL:outputURL fileType:AVFileTypeQuickTimeMovie error:&error];
   NSParameterAssert(_videoWriter);
-
+  if (error) {
+    _eventSink(@{@"event" : @"error", @"errorDescription" : error.description});
+  }
```

Comment:
Do we want to continue if there is an error (or should it return NO)?

Reply:
Best to return NO, will do.